sql/core/src/test/resources/sql-tests/inputs/describe.sql

```diff
@@ -122,6 +122,12 @@ DESC TABLE EXTENDED e;
 
 DESC FORMATTED e;
 
+CREATE TABLE f PARTITIONED BY (B, C) AS SELECT 'APACHE' A, CAST('SPARK' AS BINARY) B, TIMESTAMP'2018-11-17 13:33:33' C;
```

Comment:
does it create a hive table, shall we put `USING json` to explicitly use spark natvie data source?

Reply:
Not necessary, the default behavior w/o `USING` keyword defaults to parquet

Reply:
I think it's still better to make it explicit.

Reply:
addressed